### PR TITLE
updated after removing all independent extensions to their own repos

### PR DESCRIPTION
**Pull request by ijstokes**
This is part of #85.  No testing has been done.  Need to confirm:

* [ ] `nbwrapper` conda package builds and works OK (cross platform)
* [ ] `anaconda-notebook` conda package builds and works OK (cross platform)
* [ ] automated Anaconda Builds work

And just some basic static code review would be great.

### CODE REVIEW

File: .binstar.yml [hunk 1]
```diff
@@ -3,12 +3,16 @@ user: anaconda-notebook
...
-  # - osx-64
-  # - win-64
+  - osx-64
+  - win-64
+  - linux-32
+  - win-32
 
 engine:
   - python=2.7
+  - python=3.3
```
Comment (malev) [inline .binstar.yml]: so are we doing `3.3`?

File: .binstar.yml [hunk 2]
```diff
@@ -21,3 +25,31 @@ script:
   - conda build --no-test ./conda.recipe/
 
 build_targets: conda
+---
```
Comment (malev) [inline .binstar.yml]: is this possible???? if so, WOOOOW!

File: .binstar.yml (hunk 1 above)
Comment (mmarchetti) [inline .binstar.yml]: not at this time

File: .binstar.yml [hunk 3]
```diff
@@ -21,3 +25,31 @@ script:
...
+package: nbwrapper
```
Comment (mmarchetti) [inline .binstar.yml]: there is a .binstar.yml in the nbwrapper directory and that is where anaconda.org CI is currently pointing (via the `subdirectory` field), so I don't think this section is needed.

**Comment by mmarchetti** [comment]
There have been other PRs merged - it might be good to merge master into this branch and see what is still outstanding.

**Pull request merged**